.github/workflows/ci-cd.yml

```
@@ -311,3 +311,46 @@ jobs:
+        run: ls vacanza-holidays-${{ github.ref_name }}.zip || exit 1
+        shell: bash
+
+      - name: Upload to Zenodo
+        id: upload  # Explicitly set the step ID to match the action's output step
+        uses: ./.github/actions/zenodo-upload
+        with:
+          token: ${{ secrets.ZENODO_TOKEN }}
+          files: ${{ env.FILES }}
+          concept_id: ${{ secrets.ZENODO_CONCEPT_ID }}
+          metadata_file: "CITATION.cff"
+          sandbox: "true"  # Set to "false" for production
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)